instructor/ui/instructor.js

```diff
@@ -107,12 +107,14 @@ $(function(){
             );
             $('.jqDialog_header').remove();
             $('#jqDialog_box').prepend("<h1 class='jqDialog_header' role='heading' >Warning</h1>");
+            showed_warning=true;
```

Comment:
This will stop showing the warning whether the user clicked yes or no, but if that's your intention it's fine